android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/FormsActivity.java

```diff
@@ -14,41 +14,43 @@
 import android.widget.LinearLayout;
 import android.widget.ScrollView;
 import android.widget.Toast;
-import com.liferay.apio.consumer.cache.ThingsCache;
 import com.liferay.apio.consumer.model.Thing;
+import com.liferay.mobile.formsscreenletdemo.presenter.FormsPresenter;
+import com.liferay.mobile.formsscreenletdemo.contract.FormsViewContract;
 import com.liferay.mobile.formsscreenletdemo.util.DemoUtil;
-import com.liferay.mobile.formsscreenletdemo.util.ResourceType;
 import com.liferay.mobile.screens.base.ModalProgressBarWithLabel;
 import com.liferay.mobile.screens.thingscreenlet.screens.ThingScreenlet;
 import com.liferay.mobile.screens.thingscreenlet.screens.events.ScreenletEvents;
 import com.liferay.mobile.screens.thingscreenlet.screens.views.BaseView;
-import com.liferay.mobile.screens.thingscreenlet.screens.views.Detail;
 import com.liferay.mobile.screens.thingscreenlet.screens.views.Scenario;
 import com.liferay.mobile.screens.util.AndroidUtil;
 import com.liferay.mobile.screens.viewsets.defaultviews.ddm.events.FormEvents;
 import kotlin.Unit;
 import com.liferay.mobile.formsscreenletdemo.R;
-import com.liferay.mobile.formsscreenletdemo.util.Constants;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * @author Luísa Lima
  */
 public class FormsActivity extends AppCompatActivity
-	implements ScreenletEvents, SwipeRefreshLayout.OnRefreshListener, SwipeRefreshLayout.OnChildScrollUpCallback {
+	implements ScreenletEvents, SwipeRefreshLayout.OnRefreshListener, SwipeRefreshLayout.OnChildScrollUpCallback,
+	FormsViewContract.FormsView {
```

Comment:
You are implementing FormsView but you are always using the concrete reference FormsActivity